marconi-chain-index/src/Marconi/ChainIndex/Indexers.hs

```diff
@@ -142,10 +146,11 @@ buildIndexers securityParam catchupConfig utxoConfig mintEventConfig epochStateC
   mainCoordinator <- lift $ standardCoordinator mainLogger [blockCoordinator, chainTipWorker]
 
   let currentSyncPointIndexer =
-        CurrentSyncPoint.CurrentSyncPointQueryIndexer
-          mainCoordinator
-          blockInfoMVar
-          chainTipMVar
+        Core.withTrace mainLogger $
```

Comment:
We may want to append a logger name to `mainLogger` it to differentiate its events from the ones of the main coordinator.